lib/nxt_error_registry/version.rb

```diff
@@ -1,3 +1,3 @@
 module NxtErrorRegistry
-  VERSION = "0.2.5".freeze
+  VERSION = '0.2.5'.freeze
```

Comment:
I think you need to bump the version so that depfu can update this gem in our projects

Reply:
Yes, will do it in separate PR!